vespajlib/src/main/java/com/yahoo/tensor/functions/Concat.java

```diff
@@ -97,7 +75,7 @@ public Tensor evaluate(EvaluationContext<NAMETYPE> context) {
         IndexedTensor aIndexed = (IndexedTensor) a; // If you get an exception here you have implemented a mixed tensor
         IndexedTensor bIndexed = (IndexedTensor) b;
 
-        TensorType concatType = type(a.type(), b.type());
+        TensorType concatType = TypeResolver.concat(a.type(), b.type(), dimension);
```

Comment:
it looks like this should be some lines earlier, and then compute `combinedValueType` from `concatType`

Reply:
Thanks!